src/grammar.pest

```diff
@@ -1,6 +1,6 @@
-selector = _{ SOI ~ jsonPath ~ EOI }
+selector = _{ SOI ~ rootSelector ~ jsonPath ~ EOI }
 
-jsonPath = ${ rootSelector ~ matcher* }
+jsonPath = ${ matcher* }
```

Comment:
Now that `jsonPath` excludes the root selector, I think it needs to be renamed.

Reply:
done